xml/System.Windows.Forms/Application.xml

```diff
@@ -1,2205 +1,2225 @@
+      <MemberSignature Language="DocId" Value="M:System.Windows.Forms.Application.Run" />
+      <MemberSignature Language="VB.NET" Value="Public Shared Sub Run ()" />
+      <MemberSignature Language="F#" Value="static member Run : unit -&gt; unit" Usage="System.Windows.Forms.Application.Run " />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; static void Run();" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters />
+      <Docs>
+        <summary>Begins running a standard application message loop on the current thread, without a form.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ In a Win32-based or Windows Forms application, a message loop is a routine in code that processes user events, such as mouse clicks and keyboard strokes. Every running Windows-based application requires an active message loop, called the main message loop. When the main message loop is closed, the application exits. In Windows Forms, this loop is closed when the <xref:System.Windows.Forms.Application.Exit%2A> method is called, or when the <xref:System.Windows.Forms.Application.ExitThread%2A> method is called on the thread that is running the main message loop.  
+  
+ Most Windows Forms developers will not need to use this version of the method. You should use the <xref:System.Windows.Forms.Application.Run%28System.Windows.Forms.Form%29> overload to start an application with a main form, so that the application terminates when the main form is closed. For all other situations, use the <xref:System.Windows.Forms.Application.Run%28System.Windows.Forms.ApplicationContext%29> overload, which supports supplying an <xref:System.Windows.Forms.ApplicationContext> object for better control over the lifetime of the application.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">A main message loop is already running on this thread.</exception>
+        <altmember cref="M:System.Windows.Forms.Application.Exit" />
+        <altmember cref="M:System.Windows.Forms.Application.ExitThread" />
+        <altmember cref="M:System.Windows.Forms.Application.DoEvents" />
+      </Docs>
+    </Member>
+    <Member MemberName="Run">
+      <MemberSignature Language="C#" Value="public static void Run (System.Windows.Forms.ApplicationContext context);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig void Run(class System.
```

Comment:
This looks wonky
![image](https://user-images.githubusercontent.com/4403806/127411808-8d11f5e7-f2aa-43d0-bb01-c4c6ec227927.png)

Link: https://review.docs.microsoft.com/en-us/dotnet/api/system.windows.forms.application.setdefaultfont?view=net-6.0&branch=pr-en-us-6940#remarks